v0/destinations/af/transform.js

```diff
@@ -185,16 +185,11 @@ function processEventTypeTrack(message) {
       break;
     }
   }
-  let payload;
-  if (isUnIdentifiedEvent) {
-    payload = getEventValueForUnIdentifiedTrackEvent(message);
-  } else {
-    payload = getEventValueMapFromMappingJson(
-      message,
-      mappingConfig[category.name],
-      isMultiSupport
-    );
-  }
+  const payload = getEventValueMapFromMappingJson(
```

Comment:
`isUnIdentifiedEvent` this not used anymore

Reply:
Yes, it was useful as a check earlier, if the event was not belonging to one the declared categories of events with own mappings. Unused so removing it now.